locations/spiders/blue_bottle_cafe.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+import scrapy
+import json
+from locations.items import GeojsonPointItem
+
+class BlueBottleCafeSpider(scrapy.Spider):
+    name = "bluebottlecafe"
+    allowed_domains = ["www.bluebottlecoffee.com"]
+    start_urls = (
+        'https://bluebottlecoffee.com/api/cafe_search/fetch.json?coordinates=false&query=true&search_value=all',
+    )
+    def parse(self, response):
+        results = json.loads(response.body_as_unicode())
+        for e in results["cafes"]:
```

Comment:
Please use a descriptive name for this variable instead of just `e`. Something like `region_name`.